wat

```diff
@@ -22,6 +26,13 @@ mark_lines() {
 
 : ${PACMAN_LOG:=/var/log/pacman.log}
 
+while [ $# -gt 0 ]; do
+    case $1 in
+        -i|--installed) SHOW_INSTALLED=1; shift ;;
```

Comment:
My original intention was for it to be an env var. In the current setup, it works as both: -i|--installed will set it, or you can define it in the environment.